### PR TITLE
chore(operators): add openebs 2.12.0 release operator yamls

### PR DESCRIPTION
**Pull request by prateekpandey14**
Signed-off-by: prateekpandey14 <prateek.pandey@mayadata.io>

#### Special notes for your reviewer:

Adds
- openebs, cstor, 2.12.0 (RC1, RC2,GA)
- lvm operator 0.8.0 (RC1,RC2, GA)
- ndm operator 1.6.1 (RC1,RC2,GA)
- zfs operator 1.9.1 (RC1,RC2,GA)
- Hostpath operator (GA)
- microk8s cstor operator (GA)
- openebs operator lite (GA)
-  Jiva operator (2.12.1 GA)

### CODE REVIEW

**Comment by kmova** [comment]
@prateekpandey14 - the NDM operator changes from https://github.com/openebs/node-disk-manager/pull/612 needs to be pulled as well. ( cc: @Ab-hishek )

**Review by pawanpraka1 — APPROVED** [review]
lvm and zfs operator yamls are good.

File: 2.12.0/openebs-operator-RC1.yaml [hunk 1]
```diff
@@ -1128,6 +1128,8 @@ spec:
       - name: openebs-provisioner-hostpath
         imagePullPolicy: IfNotPresent
         image: openebs/provisioner-localpv:2.12.0-RC1
+        args:
```
Comment (kmova) [inline 2.12.0/openebs-operator-RC1.yaml:1131]: 👍

**Comment by prateekpandey14** [comment]
> the NDM operator changes from openebs/node-disk-manager#612 needs to be pulled as well. ( cc: @Ab-hishek )

@Ab-hishek can you push NDM exporter specific changes to ndm-operators.

**Comment by Ab-hishek** [comment]
Added  NDM exporter changes. @prateekpandey14 PTAL.

**Comment by kmova** [comment]
@prateekpandey14 -- please check for outdated version in the YAMLs under the parent directory. 

Like in https://github.com/openebs/charts/blob/gh-pages/openebs-amazonlinux-setup.yaml#L2

File: 2.12.0/cstor-operator.yaml [hunk 2]
```diff
@@ -0,0 +1,5471 @@
...
+---
+kind: StatefulSet
+apiVersion: apps/v1
+metadata:
+  name: openebs-cstor-csi-controller
+  namespace: openebs
+  labels:
+    name: openebs-cstor-csi-controller
+    openebs.io/component-name: openebs-cstor-csi-controller
+    openebs.io/version: 2.11.0
```
Comment (nsathyaseelan) [inline 2.12.0/cstor-operator.yaml:1054]: @prateekpandey14  can you please update the version to 2.12.0

**Review by nsathyaseelan — CHANGES_REQUESTED** [review]

**Comment by Ab-hishek** [comment]
> Added NDM exporter changes. @prateekpandey14 PTAL. Do I also need to add them RC1 and RC2 operator yamls also. For now only added them to ndm-operator yaml.

Have updated the ndm RC1 and RC2 operatoe yamls with ndm exporter changes.

File: 2.12.0/cstor-operator.yaml (hunk 2 above)
Comment (prateekpandey14) [reply 2.12.0/cstor-operator.yaml]: fixed

**Review by nsathyaseelan — APPROVED** [review]
lgtm

**Review by w3aman — APPROVED** [review]
Zfs and lvm operator yamls -- /LGTM

File: openebs-amazonlinux-setup.yaml [hunk 3]
```diff
@@ -6,7 +6,7 @@ metadata:
   name: openebs-amazonlinux-init
   labels:
     openebs.io/component-name: openebs-amazonlinux-init
-    openebs.io/version: 1.4.0
+    openebs.io/version: 2.12.0
```
Comment (kmova) [inline openebs-amazonlinux-setup.yaml:9]: In this file, the change required is to replace the `apiVersion`.

**Pull request merged**